R/news.R

```diff
@@ -9,13 +9,23 @@
 #' @export
 use_news_md <- function(open = rlang::is_interactive()) {
   check_is_package()
-  desc <- proj_desc()
+
+  ver <- proj_version()
+  on_cran <- !is.null(cran_version())
+  version_string <- if (is_dev_version(ver)) "(development version)" else ver
+
+  if (!on_cran && package_version(ver) <= package_version("0.1.0")) {
```

Comment:
This choice of version number was a bit arbitrary... maybe we could just condition on whether or not it's on CRAN? I know that's not everybody's final destination but I think most of this and related tooling is CRAN-oriented...

Reply:
Great, I like all of that